src/script/lua_api/l_http.cpp

```diff
@@ -23,6 +23,7 @@ with this program; if not, write to the Free Software Foundation, Inc.,
 #include "lua_api/l_http.h"
 #include "httpfetch.h"
 #include "settings.h"
+#include "debug.h"
```

Comment:
Well, `FATAL_ERROR_IF` is defined in `debug.h`. Technically this is not neccesary, because some other header files also include `debug.h`, but I think it's cleaner to add this header since the code depends on it.